src/utils/safeHref.ts

```diff
@@ -3,6 +3,7 @@ export interface URLInfo {
     sameOrigin: boolean
 }
 
+const PROTOCOL_REGEX = /^[a-z]+:/
```

Comment:
Is there any chance of this falsely matching relative paths (protocol-less hrefs)?